hub/policy/src/integration-test/java/uk/gov/ida/integrationtest/hub/policy/StateControllerFactoryTest.java

```diff
@@ -12,36 +11,58 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.Mock;
-import uk.gov.ida.hub.shared.eventsink.EventSinkProxy;
-import uk.gov.ida.hub.policy.configuration.PolicyConfiguration;
 import uk.gov.ida.hub.policy.PolicyModule;
+import uk.gov.ida.hub.policy.configuration.PolicyConfiguration;
 import uk.gov.ida.hub.policy.domain.AbstractState;
 import uk.gov.ida.hub.policy.domain.StateController;
 import uk.gov.ida.hub.policy.domain.StateTransitionAction;
-import uk.gov.ida.hub.policy.domain.controller.*;
```

Comment:
You're getting into religious war territory with that question.

The short answer is that our style guide says so: https://gds-way.cloudapps.digital/manuals/programming-languages/java.html#imports

The longer answer is that it can easily lead unintended name clashes, especially when some packages have lots of very generically named classes.